context/app/static/js/components/workspaces/WorkspacesList.jsx

```diff
@@ -69,12 +73,21 @@ function WorkspacesList() {
                 Created {workspace.datetime_created.slice(0, 10)}
                 <button
                   type="submit"
+                  disabled={workspace.jobs.length > 0 /* TODO: And/or check workspace status? */}
```

Comment:
@jpuerto-psc -- This list of jobs is already filtered down; See below. This should probably also be disabled if the workspace is already in the process of deletion... Do any other cases come to mind? The API docs don't list the possible statuses... is it just `deleting`?

Reply:
Correct, the workspace status can be one of the following:

- idle
- active
- deleting

I would, for now, check for job length as this will be more accurate than checking for a workspace's status. So far I have seen these two match up 100% but since they are updated asynchronously it is possible that they might be de-synced.